### DISCUSSION TITLE
Welcome to PassGen Discussions!

### DISCUSSION THREAD

**Discussion by Alireza-Razavi** [Alireza-Razavi/PassGen#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by x100dev** [comment]
This is a comment :)

**Reply by Alireza-Razavi** [reply]
OK this is reply so :)

**Comment by Alireza-Razavi** [comment]
Also this is another comment :)
